lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -311,11 +311,23 @@ function reportAssertion(assertion) {
  * @return {{passed: number, failed: number}}
  */
 function report(results) {
+  let correctCount = 0;
+  let failedCount = 0;
+
   reportAssertion(results.finalUrl);
+  if (results.finalUrl.equal) {
+    correctCount++;
+  } else {
+    failedCount++;
+  }
+
   reportAssertion(results.errorCode);
+  if (results.errorCode.equal) {
+    correctCount++;
+  } else {
+    failedCount++;
+  }
 
-  let correctCount = 0;
-  let failedCount = 0;
   results.audits.forEach(auditAssertion => {
```

Comment:
```suggestion
  [results.finalUrl, results.errorCode, ...results.audits].forEach(auditAssertion => {
```

Reply:
could do this instead and drop the two reportAssertions above.

Reply:
nice

do we want to continue showing green ticks for passing? this is what it looks like  with those changes + moving `reportAssertion` out of the if statement:

![image](https://user-images.githubusercontent.com/4071474/52662772-e4904b80-2eb9-11e9-80bf-f44813d36a32.png)


Keeping `reportAssertion` where it is-

![image](https://user-images.githubusercontent.com/4071474/52662809-f96cdf00-2eb9-11e9-81d9-aee97daa986d.png)

Reply:
i like this best

![image](https://user-images.githubusercontent.com/4071474/52663149-d68efa80-2eba-11e9-9a42-68a591e41246.png)